tests/unittests/test_cli.py

```diff
@@ -218,18 +219,27 @@ def test_wb_devel_schema_subcommand_parser(self):
             self.stderr.getvalue())
 
     def test_wb_devel_schema_subcommand_doc_content(self):
-        """Validate that doc content is sane from known examples."""
+        """Validate that doc content is sane from known examples.
```

Comment:
Since we are touching this line, let's update inclusive wording where we can.
```suggestion
        """Validate that doc content from known examples is well-structured.
```

or coherent or correct etc.